docs/extensions/controlplane-webhooks.md

```diff
@@ -70,7 +70,7 @@ The `env` field of the `kube-apiserver` container **shall not** contain any prov
 
 The `volumes` field of the pod template of the `kube-apiserver` deployment, and respectively the `volumeMounts` field of the `kube-apiserver` container **shall not** contain any provider-specific `Secret` or `ConfigMap` resources. If such resources should be mounted as volumes, this should be done by webhooks.
 
-The `kube-apiserver` `Service` **may** be of type `LoadBalancer`, but **shall not** contain any provider-specific annotations that may be needed to actually provision a load balancer resource in the Seed provider's cloud. If any such annotations are needed, they should be added by webhooks (typically `controlplaneexposure` webhooks).
+The `kube-apiserver` `Service` **may** be of type `LoadBalancer`, but **shall not** contain any provider-specific annotations that may be needed to actually provision a load balancer resource in the Seed provider's cloud. If any such annotations are needed, they should be added by webhooks (typically `seedprovider` webhooks).
 
 The `kube-apiserver` `Service` **will** be of type `ClusterIP`. In this case, Gardener **will** label this `Service` with `core.gardener.cloud/apiserver-exposure: gardener-managed` label (deprecated, the label will no longer be added as of `v1.80`) and expects that no mutations happen.
```

Comment:
The docs are quite outdated. The kube-apiserver can be only of type `ClusterIP` after the APIServerSNI/ManagedIstio feature gates are unconditionally enabled.
The  `core.gardener.cloud/apiserver-exposure: gardener-managed` label seems to be removed already. The provider extensions don't mutate the kube-apiserver Service anymore. Looks like the whole section needs some rework.